outlierdetect/__init__.py

```diff
@@ -12,5 +12,5 @@
 # or implied.  See the License for the specific language governing permissions and limitations
 # under the License.
 
-
-from outlierdetect import run_sva, run_mma, __doc__
+# Leading to a circular import, so commented out.
+# from outlierdetect import run_sva, run_mma, __doc__
```

Comment:
This is not the correct solution. More details here:

https://docs.python.org/3/reference/import.html#regular-packages

Correct code should be:
```python
from .outlierdetect import run_sva, __doc__
# run_mma will be missing when stats are not available
try:
    from .outlierdetect import run_mma
except ImportError:
    pass
```

I've tested this on 3.8.6 and 2.7 and it works on both